### PR TITLE
:memo: added analysis options and refactor code to follow analysis options.

### PR DESCRIPTION
**Pull request by ParthBaraiya**

### CODE REVIEW

File: lib/src/week_view/_internal_week_view_page.dart [hunk 1]
```diff
@@ -104,15 +105,11 @@ class InternalWeekViewPage<T> extends StatelessWidget {
       height: height + weekTitleHeight,
       width: width,
       child: Column(
-        mainAxisSize: MainAxisSize.max,
-        mainAxisAlignment: MainAxisAlignment.start,
```
Comment (DevarshRanpara) [inline lib/src/week_view/_internal_week_view_page.dart:108]: Why we removed `MainAxisSize` and `MainAxisAlignment`?

File: lib/src/week_view/_internal_week_view_page.dart [hunk 2]
```diff
@@ -104,15 +105,11 @@ class InternalWeekViewPage<T> extends StatelessWidget {
...
         crossAxisAlignment: CrossAxisAlignment.end,
         children: [
           SizedBox(
             width: width,
             child: Row(
-              mainAxisSize: MainAxisSize.max,
-              mainAxisAlignment: MainAxisAlignment.start,
               crossAxisAlignment: CrossAxisAlignment.start,
```
Comment (DevarshRanpara) [inline lib/src/week_view/_internal_week_view_page.dart:113]: Why we removed `MainAxisSize` and `MainAxisAlignment`?

File: lib/src/components/month_view_components.dart [hunk 3]
```diff
@@ -116,9 +116,6 @@ class FilledCell<T> extends StatelessWidget {
     return Container(
       color: backgroundColor,
       child: Column(
-        mainAxisAlignment: MainAxisAlignment.start,
-        crossAxisAlignment: CrossAxisAlignment.center,
-        mainAxisSize: MainAxisSize.max,
         children: [
```
Comment (DevarshRanpara) [inline lib/src/components/month_view_components.dart:119]: Why we removed `MainAxisSize`, `MainAxisAlignment` and `CrossAxisAlignment`?

File: lib/src/components/month_view_components.dart [hunk 4]
```diff
@@ -149,8 +146,6 @@ class FilledCell<T> extends StatelessWidget {
                   physics: BouncingScrollPhysics(),
                   child: Column(
                     mainAxisSize: MainAxisSize.min,
-                    crossAxisAlignment: CrossAxisAlignment.center,
-                    mainAxisAlignment: MainAxisAlignment.start,
```
Comment (DevarshRanpara) [inline lib/src/components/month_view_components.dart:153]: Why we removed `MainAxisAlignment` and `CrossAxisAlignment`?

File: lib/src/components/month_view_components.dart [hunk 5]
```diff
@@ -166,7 +161,6 @@ class FilledCell<T> extends StatelessWidget {
                           padding: const EdgeInsets.all(2.0),
                           alignment: Alignment.center,
                           child: Row(
-                            mainAxisSize: MainAxisSize.max,
```
Comment (DevarshRanpara) [inline lib/src/components/month_view_components.dart:169]: Why we removed `MainAxisSize`?

**Review by DevarshRanpara — COMMENTED** [review]
@ParthBaraiya I have seen, we removed `MainAxisSize`, `MainAxisAlignment` and `CrossAxisAlignment`,
Could you let me know why we made those changes?

File: lib/src/week_view/_internal_week_view_page.dart (hunk 1 above)
Comment (ParthBaraiya) [reply lib/src/week_view/_internal_week_view_page.dart:108]: Values we provided as arguments are default values for Column. So whether we provide it or not doesn't matter.

File: lib/src/week_view/_internal_week_view_page.dart (hunk 2 above)
Comment (ParthBaraiya) [reply lib/src/week_view/_internal_week_view_page.dart:113]: Values we provided as arguments are default values for Row. So whether we provide it or not doesn't matter.

File: lib/src/components/month_view_components.dart (hunk 3 above)
Comment (ParthBaraiya) [reply lib/src/components/month_view_components.dart:119]: Values we provided as arguments are default values for Column. So whether we provide it or not doesn't matter.

File: lib/src/components/month_view_components.dart (hunk 4 above)
Comment (ParthBaraiya) [reply lib/src/components/month_view_components.dart:153]: Values we provided as arguments are default values for Column. So whether we provide it or not doesn't matter.

File: lib/src/components/month_view_components.dart (hunk 5 above)
Comment (ParthBaraiya) [reply lib/src/components/month_view_components.dart:169]: Values we provided as arguments are default values for Column. So whether we provide it or not doesn't matter.

**Review by DevarshRanpara — APPROVED** [review]
LGTM

**Pull request merged**